base/include/SoundRecord.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "Module.h"
+
+class SoundRecordProps : public ModuleProps
+{
+public:
+    SoundRecordProps(
+        int _sampleRate,
+        int _channel,
+        int _device,
+        int _proccessingRate) : sampleRate(_sampleRate),
+                                channel(_channel),
+                                device(_device),
```

Comment:
How does the user of this class know which number to be passed for "device" ? a.k.a is there a better name for this argument ?